projects/plugins/jetpack/modules/videopress/class.videopress-gutenberg.php

```diff
@@ -37,6 +38,7 @@ private function __construct() {
 		add_action( 'jetpack_register_gutenberg_extensions', array( $this, 'set_extension_availability' ) );
 		add_action( 'enqueue_block_editor_assets', array( $this, 'override_video_upload' ) );
 		add_action( 'enqueue_block_editor_assets', array( $this, 'add_resumable_upload_support' ) );
+		Block_Replacement::init();
```

Comment:
I'm not sure if it was a good place to include this call.

The other option was to add a new class that basically only does this call. However, I needed to include it in other parts as well to enable the module.

Since `Block_Replacement` class partially substitutes another Gutenberg block, I considered it a reasonable trade-off.